woocommerce/woocommerce-android
# Domain change: Access to domains

This is a pretty simply PR, which just adds an additional constraint to the access to Domains settings. The reason is that only  the Administrator user role has the permission to access these APIs.

**To test:**
1. Create a user with a different role than the Admin (Shop manager, for example) on an Atomic site
2. Log in using this user
3. Go to Settings and verify the Domains setting is not visible
4. Log out and log in as an Admin
5. Go to Settings and verify the Domains setting is there
